modules/aws_ecs_fargate/main.tf

```diff
@@ -0,0 +1,247 @@
+terraform {
+  required_providers {
+    aws = {
+        source = "hashicorp/aws"
+        version = "~> 4.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.aws_region
+}
+
+resource "aws_ecs_cluster" "this" {
+  name = "${var.deployment_name}-ecs"
+
+  setting {
+    name  = "containerInsights"
+    value = var.ecs_insights_enabled
+  }
+}
+
+resource "aws_cloudwatch_log_group" "this" {
+  name              = "${var.deployment_name}-ecs-log-group"
+  retention_in_days = var.log_retention_in_days
+}
+
+resource "aws_db_instance" "this" {
```

Comment:
probably should allow people to opt-out of creating a DB with some var and then support the customer passing in the db information through env vars as normal